code/game/objects/effects/alien_acid.dm

```diff
@@ -18,7 +18,8 @@
 
 	if(acid_amt)
 		acid_level = min(acid_amt*acid_pwr, 12000) //capped so the acid effect doesn't last a half hour on the floor.
-
+		if(acid_level < 0)//Fix for a weird runtime.
```

Comment:
Whatever is causing the acid runtimes, you need to `CLAMP(round(acid_value, 1), 0, maximum acid amount/99999)`